.lando.yml

```diff
@@ -49,6 +51,10 @@ services:
         DB_USER_DRUPAL: drupal9
         DB_PASS_DRUPAL: drupal9
         DB_HOST_DRUPAL: database
+        # Support debugging with XDEBUG 3.
+        XDEBUG_MODE:
```

Comment:
`XDEBUG_MODE` is mapped to `xdebug` key value in `.lando.yml`: `XDEBUG_MODE: (options.xdebug === false) ? 'off' : options.xdebug,`. Also, `if (options.xdebug === true) options.xdebug = 'debug';`.

Source: https://github.com/lando/lando/blob/master/plugins/lando-services/services/php/builder.js#L147-L158